src/utils/ecommerce/psp.ts

```diff
@@ -135,7 +135,7 @@ const createSuccessGetPspByPaymentMethodsIdResponseEntityBelowThresholdV2 = ():
       abi: "03069",
       bundleDescription:
         "Clienti e non delle Banche del Gruppo Intesa Sanpaolo possono disporre pagamenti con carte di pagamento VISA-MASTERCARD",
-      bundleName: "Intesa Sanpaolo S.p.A",
+      bundleName: "Intesa Sanpaolo",
```

Comment:
what about disable linting against those mocked responses? it could be acceptable to have duplicated strings in mocked responses

Reply:
Done in [c8e80ce](https://github.com/pagopa/pagopa-checkout-be-mock/pull/85/commits/c8e80ce4512f79ff51a762adcdd9237c58928bde)